src/writer.jl

```diff
@@ -0,0 +1,309 @@
+
+"""
+    write(path::AbstractString, handle::Shapefile.Handle)
+    write(path::AbstractString, geoms)
+
+Write geometries to file. `geoms` can be a table or any iterable of GeoInterface.jl
+combatible geometry objects, with `missing` values allowed.
+
+Note: As DBFTables.jl does not yet write, we can't write Table or FeatureCollection
+data besides geometries. Only .shp and .shx files are written currently.
+"""
+write(path::AbstractString, h::Handle) = write(path, h.shapes)
+function write(path::AbstractString, obj; force=false)
+    paths = _shape_paths(path)
+
+    # File existance check
+    if isfile(paths.shp)
+        if force
+            rm(paths.shp)
+        else
+            throw(ArgumentError("File already exists at `$(paths.shp)`. Use `force=true` to write anyway."))
+        end
+    end
+
+    # Handle tabular data
+    if Tables.istable(obj)
+        geomcol = GI.geometrycolumn(obj)
```

Comment:
Note that it's `geometrycolumns(featurecollection) = (:geometry,)`, so +s, and you need to handle the tuple.

Reply:
Fixed already ;)